docs/skopeo-sync.1.md

```diff
@@ -88,6 +88,9 @@ Print usage statement.
 
 **--retry-times**  the number of times to retry, retry wait time will be exponentially increased based on the number of failed attempts.
 
+**--ignore**
```

Comment:
Please update `completions/bash/skopeo` as well.

Reply:
Done. Updated the PR with this change.